### DISCUSSION TITLE
跌跌不休的特斯拉

### DISCUSSION THREAD

**Discussion by wangtyphoon** [1101-GS4719-C/Python-Community-GS4719-C#976] [General]
特斯拉縱使進入2022年後，遭遇嚴重拋售，至今跌幅高達69%，依舊是韓國散戶的主要標的，今年韓國散戶累積淨購入28億美金的特斯拉股票，如果下跌的趨勢不變，這群投資人可能很快就會成為隱性經濟動盪的隱憂，且隨著特斯拉車價的日益走跌，毛利率相比於同業，也沒有明顯的優勢，失去造夢能力的馬斯克還能帶領特斯拉走向另外一次的高峰嗎?
以我個人的看法而言，馬斯克絕對是最有想法和企圖心的企業家，但營利與長久維持公司的穩定，可能不是其所擅長的!
https://ec.ltn.com.tw/article/breakingnews/4168001

**Comment by harrylee1971** [comment]
感覺馬斯克是一個能被利益驅使而去做事情的年輕人，而且他好像是負二代(?

**Comment by s96015welson** [comment]
雖然馬斯克常常放大話，不過他真的在推動科技進步上很積極，且在許多不同領域都有傑出的貢獻與突破。扣除掉喜歡炒作的部分，我滿欣賞他突破性思維，以及為產業帶來新風潮的部分，例如若是沒有Tesla，電動車產業應該會少了許多的競爭與創新。

**Comment by ghost** [comment]
我覺得買股票不該只看短期起伏，更要看長遠的走勢，看看這個產業或是公司在未來的規劃上，是否有前景，所以縱使現在是在下跌，只要有前景，也還是有投資的潛力的。
